src/structures/MessageMedia.js

```diff
@@ -98,6 +98,32 @@ class MessageMedia {
 
         return new MessageMedia(mimetype, res.data, filename);
     }
+    
+    /** 
+     * Securely saves the data to a file 
+     * @paranm {string} filePath the path to where you want to save the file (no extensions)
+     * @example <caption>Example usage of this function</caption>
+     * const media = await msg.downloadMedia();
+     * media.toFilePath('/home/purpshell/Documents/message') // the code adds the extension for you (if you enter a directory we will use the filename property (if it's null then we'll throw an error))
+     * @returns {null} no need to return anything
+     */
+    async toFilePath(filePath) {
+        const ext = mime.getExtension(this.mimetype);
+        if (fs.existsSync(filePath)) {
+            const stat = await fs.promises.stat(filePath);
+            if (stat.isDirectory()) {
+                if (this.filename){
+                    filePath += this.filename;
```

Comment:
Wouldn't this break if the input doesn't end with a folder separator ?

Like with input `/path/to/folder` it'll save to `/path/to/folderfilename.ext`

Reply:
aight